specification/httpx/src/main/scripts/org/kaazing/specification/httpx/opening/upgrades.with.handshake.protocol/request.rpt

```diff
@@ -0,0 +1,21 @@
+connect http://localhost:8000/echo
+connected
+
+write method "GET"
+write version "HTTP/1.1"
+write header host
+write header "Upgrade" "websocket"
+write header "Connection" "Upgrade"
+write header "Sec-WebSocket-Key" "dGhlIHNhbXBsZSBub25jZQ=="
+write header "Sec-WebSocket-Version" "13"
+write header "Sec-WebSocket-Protocol" "x-kaazing-handshake"
+
+read status "101" /.+/
+read version "HTTP/1.1"
+read header "Upgrade" "websocket"
+read header "Connection" "Upgrade"
+read header "Sec-WebSocket-Accept" "s3pPLMBiTxaQ9kYGzzhZRbK+xOo="
+read header "Sec-WebSocket-Protocol" "x-kaazing-handshake"
+read header "Server" /.+/
+read header "Date" /.+/
+

```

Comment:
I think we should rename `upgrades.wth.handshake.protocol` to `extended.handshake.protocol.negotiated`